OneCommunityGlobal/HighestGoodNetworkApp
# Jatin create different question sets

# Description
Please include the exact bug/functionality description and a summary of the changes/ related issues. Please also include any other relevant motivation and context:
<img width="570" alt="339" src="https://github.com/user-attachments/assets/9fbd9784-28a7-477a-b3dc-9a304a40a01f">


## Related PRS (if any):
This frontend PR is related to the [#1167](https://github.com/OneCommunityGlobal/HGNRest/pull/1167) backend PR.

## Main changes explained:
- src/components/JobFormBuilder.jsx file A for creating new job forms.

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to route `/jobformbuilder'
6. Verify Functionality as described in the video.

## Screenshots or videos of changes:
https://github.com/user-attachments/assets/7f9f9f94-8a4e-477d-aeb2-f13b80194d7e


## Note:
This PR is created for the job Application Page. This page needs to be linked to a page which is yet to made by another volunteer, which is why in the video all the API's are being tested that way.
